### DISCUSSION TITLE
versao 4.35.1

### DISCUSSION THREAD

**Discussion by ernandescardoso** [RamonSilva20/mapos#1517] [General]
boa noite em todas as telas onde clik aparece um pop up com a seguinte msg 

Sucesso!
Configurações do sistema atualizadas com sucesso!

onde devo corrigir , caso seja possive
![pop up](https://user-images.githubusercontent.com/57164421/136495269-ae18b3d5-1bf2-4628-982a-69f632192bbd.png)
l.

**Comment by guskid** [comment]
Já tive este problema quando estava usando lubuntu com servidor php. Unica maneira que "resolvi" foi tirando o swetalert do código raiz.

**Reply by andremp15** [reply]
Mas eu estou utilizando hospedagem da hostgator

**Comment by andremp15** [comment]
Boa tarde a todos, fiz atualização da versão 4.39 a mais recente porém, ao faturar uma ordem de serviço simplesmente não fatura e quando fatura no outro dia some mostrando que não está faturado mas consta faturado conforme a imagem abaixo como eu faço para corrigir esse erro

![Screenshot_20230120_224512_Chrome.jpg](https://user-images.githubusercontent.com/117945999/213870916-96f04494-6d12-4d6b-9855-29d24a41eca5.jpg)

**Reply by andremp15** [reply]
Mas eu estou usando servidor da hostgator

**Reply by Buceli** [reply]
eu Também estou com o mesmo problema.

**Comment by CarlosCavalcante87** [comment]
![Screenshot_2024-04-08-23-12-18-133_com android chrome-edit](https://github.com/RamonSilva20/mapos/assets/76854766/2abfa896-623a-4892-aeb3-14922bb8860c)
Está tela deveria ser trocada por uma que exibir se os serviços e produtos mais vendidos no mês. 
Sugestão 🤝☺️

**Comment by ThaynanGarces** [comment]
Boa noite, na versão 4.53.1 quando mudo para o modo mobile alguns campos de edição estão sem legendas ou especificações do que deveria ser digitado no campo

**Reply by willph** [reply]
Se de poderia postar o screenshot, por favor?

**Reply by ThaynanGarces** [reply]
> Se de poderia postar o screenshot, por favor?

![IMG_20260211_224732.jpg](https://github.com/user-attachments/assets/6183d243-53b7-47f0-9e18-4c20181cbd92)

**Reply by willph** [reply]
Ok.

Obrigado por relatar.

**Reply by ThaynanGarces** [reply]
O layout na parte do login do cliente, está um pouco sem responsividade!

![IMG_20260211_225156.jpg](https://github.com/user-attachments/assets/a4e045de-d9c2-421b-a442-2bfdf75c3619)

![IMG_20260211_225222.jpg](https://github.com/user-attachments/assets/9d6d8183-52e8-4251-8dd5-6b1e513c0094)
